guide/how-it-works/stabilizing-bitcoin-value.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: guide
+title: Stabilizing bitcoin value
+description:
+nav_order: 13
+parent: How it works
+permalink: /guide/how-it-works/stabilizing-bitcoin-value/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/stabilizing-bitcoin-value/stabilizing-bitcoin-value-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+Link to diagram design source:
+https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?type=design&node-id=3481%3A1054&mode=design&t=k18mAClMdVyoDxbG-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/stabilizing-bitcoin-value.jpg"
+   retina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/stabilizing-bitcoin-value@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/stabilizing-bitcoin-value-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/stabilizing-bitcoin-value-mobile@2x.jpg"
+   alt-text = "Diagram of bitcoin being swapped for euro and synthetic euro"
+   width = 1600
+   height = 400
+   layout = "full-width"
+%}
+
+# Stabilizing bitcoin value
+{:.no_toc}
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+The price of bitcoin can wildly fluctuate relative to government-issued currencies (also called [fiat currencies](https://bitcoin.design/guide/glossary/#fiat)). For a user whose primary use is building up savings over the long-term, this might not be an issue. For example, they may be more interested in the price of bitcoin relative to the Euro in 10 years rather than the price tomorrow morning.
+
+However, for a user who hopes to use bitcoin for daily spending or remittances, the volatility is a problem. Suppose Alice lives paycheck to paycheck. If she is paid in bitcoin, even a relatively small decrease in the price could make her unable to pay her Euro-denominated bills each month.
+
+There are times when we prefer to have a stable amount of bitcoin. Other times we prefer a stable amount of another currency (which means that the amount of bitcoin fluctuates in relation).
+
+The most direct and established option is for Alice to simply trade her bitcoin for local currency via an exchange. However, there are various techniques that help to stabilize bitcoin to the dollar value without selling. Some of these techniques are:
+
+- [Tokens](#tokens)
+- [Synthetic currencies](#synthetic-currencies)
+- [Discrete Log Contracts (DLCs)](#discrete-log-contracts-dlcs)
```

Comment:
There is one more category which you could write about: Stable Channels see Tony's tweet here: https://x.com/tonklaus/status/1743179494524121559?s=20

Reply:
Thank you. I'll open that up as a separate issue once this PR is merged.